python/dgl/transforms/functional.py

```diff
@@ -2885,15 +2887,21 @@ def sort_csr_by_tag(g, tag, tag_offset_name='_TAG_OFFSET'):
     """
     if len(g.etypes) > 1:
         raise DGLError("Only support homograph and bipartite graph")
+    assert tag_type in ['node', 'edge'], "tag_type should be either 'node' or 'edge'."
+    if tag_type == 'node':
+        _, dst = g.adj(scipy_fmt='csr').nonzero()
+        tag = tag[dst]
```

Comment:
use our own `g.edges()` for getting src/dst

Reply:
`g.edges()` and `g.adj()` return different values. As it's sort on CSC/csr, seems we need to pass in csr/csc orders, instead of `g.edges()` which returns according to edge IDs.

Reply:
How about specifying the `order` argument when calling `g.edges`? https://docs.dgl.ai/generated/dgl.DGLGraph.edges.html#dgl.DGLGraph.edges

Reply:
NO, `g.edges(order='srcdst')` returns edges according to src IDs and dst IDs. But `g.adj()` returns in rows for `csr` or columns in `csc`. They are still different. Right?